application/controllers/SettingsController.php

```diff
@@ -438,20 +438,32 @@ private function checkSnmpTrapd()
           return array(1,'Port UDP/162 is not open : is snmptrapd running?');
       }
       $psOutput=array();
-      exec('ps --no-headers -o command -C snmptrapd',$psOutput);
-      if (count($psOutput) == 0)
+      $selinux_state = '';
+      if(is_executable('/usr/sbin/getenforce'))
```

Comment:
In installer.sh, it is just `getenforce` without path :
1) Can getenforce be in a different path depending on linux distro ?
2) Maybe also try getenforce without path ?

Reply:
The limitation is paths containing `sbin` are not present in the search path when running under the web server user, so `getenforce` on its own will fail. When running `installer.sh` as root, however, `/usr/sbin` is included in `$PATH`.

I have checked CentOS 6 / 7 / 8 and it's consistently `/usr/sbin/getenforce`, so RHEL/Fedora should be fine. If an Ubuntu user removes apparmor and installs selinux, the path is still `/usr/sbin/getenforce` there. I could dive into more obscure distros (openSUSE, Hardened Gentoo) if you like.